src/editor/adapters/change-signature-webview/createChangeSignatureWebviewTemplate.test.ts

```diff
@@ -19,6 +19,38 @@ describe("Change signature Webview Content", () => {
     expect(params[1].textContent).toBe("paramB");
   });
 
+  it("Should render disabled inputs", () => {
```

Comment:
🍰 In my understanding, this is more like:

```suggestion
  it("Should disable existing params inputs", () => {
```

Right?